wgs.yaml

```diff
@@ -56,6 +56,66 @@ workinggroups:
   - name: katib
     owners:
     - https://raw.githubusercontent.com/kubeflow/katib/master/OWNERS
+- dir: wg-pipelines
+  name: Pipelines
+  mission_statement: >
+    The goal of the Kubeflow Pipelines project is to build a Kubernetes optimized stack for creating and deploying ML Pipelines and enabling a rich ecosystem of pipelines components and tools. ML Pipelines project has the goal of enabling simple and reliable productionalization of autonomous ML workflows, advancing the standards of ML Engineering and making them available to a large community of users across various Kubernetes deployment options.
+
+  charter_link: charter.md
+  label: area/wg-pipelines
+  leadership:
+    chairs:
+    - github: paveldournov
+      name: Pavel Dournov
+      company: Google
+    - github: karkumar
+      name: Karthik Ramachandran
+      company: Google
+    - github: james-jwu
+      name: James Wu
+      company: Google
+    tech_leads:
+    - github: neuromage
+      name: Ajay
+      company: Google
+    - github: rmgogogo
+      name: Renmin
+      company: Google
+    - github: Bobgy
+      name: Yuan Gong
+      company: Google
+  meetings:
+  # < Needs to be added>
+  # - description: Kubeflow AutoML Working Group Meeting (Asia & Europe friendly)
+  #   day: Wednesday
+  #   time: 10:00am
+  #   tz: UTC (Coordinated Universal Time)
+  #   frequency: monthly - second Wednesday every month
+  #   archive_url: https://docs.google.com/document/d/1MChKfzrKAeFRtYqypFbMXL6ZIc_OgijjkvbqmwRV-64
+  # - description: Kubeflow AutoML Working Group Meeting (US friendly)
+  #   day: Wednesday
+  #   time: 4:00pm
+  #   tz: UTC (Coordinated Universal Time)
+  #   frequency: monthly - fourth Wednesday every month
+  #   archive_url: https://docs.google.com/document/d/1MChKfzrKAeFRtYqypFbMXL6ZIc_OgijjkvbqmwRV-64
+  contact:
+    #slack:
+    mailing_list: https://groups.google.com/forum/#!forum/kubeflow-discuss
+    teams:
+    - name: wg-pipelines-bugs
+      description: Bug Triage and Troubleshooting    
+    - name: wg-pipelines-reviews
+      description: API Changes and Reviews (API Machinery APIs, NOT all APIs)
+  subprojects:
+  - name: kfp-tekton
```

Comment:
I feel like it's better to have some chairs or leads from this project.

Reply:
Jeff means "kfp-tekton". Following the structure, I updated Pavel's PR to add Animesh to owners list for easy contacting.
https://github.com/kubeflow/community/blob/3e8bb28c49f872bd8395a7c5887b6516bbaab14c/generator/app.go#L100

"make generate" to refresh the auto-generated markdown file will be handled later after LGTM.

Reply:
thanks @rmgogogo 
Additionally, since the overall leadership should reflect the leadership from sub-projects, it would be good have a representation either in chairs or leads section at top.

Reply:
done

Reply:
so here mark as resolved. I will help take care this PR for better format, submit etc. The people list will be shorted by alphabet order per the tool.